src/test/scala/beam/agentsim/agents/PersonAgentSpec.scala

```diff
@@ -860,4 +857,14 @@ class PersonAgentSpec
     super.afterAll()
   }
 
+  override def afterEach(): Unit = {
+    import scala.language.postfixOps
+    if (iteration != null) {
```

Comment:
nitpick: we're in Scala, maybe better to handle it via `Option` type?

Reply:
Done